pages/over-ori-a.md

```diff
@@ -9,25 +9,29 @@ Hieronder wordt het belang van een archiefstandaard voor zulke informatie uitgel
 
-ORI-A heeft een vergelijkbaar doel als het ORI-informatiemodel: het gestandaardiseerd beschikbaar stellen van raadsinformatie. Deze standaardisatieslag is hard nodig, omdat ieder RIS-systeem momenteel een eigen, niet-publiekelijk gedocumenteerd formaat voor raadsinformatie hanteert. Hiermee komt de toekomstige **vindbaarheid** en **interpreteerbaarheid** van raadsinformatie in gevaar.
+ORI-A heeft hetzelfde hoofddoel als ORI: het gestandaardiseerd beschikbaar stellen van raadsinformatie. Deze standaardisatieslag is hard nodig, omdat ieder RIS-systeem momenteel een eigen, niet-publiekelijk gedocumenteerd formaat voor raadsinformatie hanteert. Hiermee komt de toekomstige **vindbaarheid** en **interpreteerbaarheid** van raadsinformatie in gevaar.
 
-Waar ORI-A verschilt van de andere initiatieven is dat het zich ook richt op het migreren van raadsinformatie. Hierin is eerst gekeken naar het ORI-informatiemodel, maar deze kon niet aan alle behoeften van archiefdiensten voldoen. Toen is besloten een archiefvariant van ORI te ontwikkelen. Je kan hier meer over lezen in [Waarom een speciale archiefstandaard?](faq)
+De ORI-standaard van de VNG voldeed echter niet volledig aan de behoeften van archiefdiensten. Hierom is besloten een archiefvariant van ORI te ontwikkelen. Je kan hier meer over lezen in [Waarom een speciale archiefstandaard?](faq)
 
 # Beheer van de standaard
-De Werkgroep Archivering Raadsinformatie beheert momenteel ORI-A. De komende tijd richten we ons op het promoten en tijdelijk ondersteunen van het gebruik van de standaard. 
+
+De [Werkgroep Archivering Raadsinformatie](colofon) beheert momenteel ORI-A. De komende tijd richten we ons op het promoten en tijdelijk ondersteunen van het gebruik van de standaard. 
 
 ## Status
 
-Het ORI-A XML-schema is momenteel in bèta, en [kan hier gedownload worden](https://ori-a.nl/downloads#xml-schema). De website ondergaat de komende tijd een publieksreview. Vanaf het najaar volgen enkele pilotimplementaties, waarin de standaard wordt ingezet in de praktijk. Updates en nieuws over ORI-A zijn te lezen op deze website of op [KIA](https://kiacommunity.nl/groups/86-videotulen/welcome).
+Het [ORI-A XML-schema](downloads#xml-schema) is momenteel in bèta. De website ondergaat de komende tijd een publieksreview. Vanaf het najaar volgen enkele pilotimplementaties, waarin de standaard wordt ingezet in de praktijk. Updates en nieuws over ORI-A zijn te lezen op deze website of op [KIA](https://kiacommunity.nl/groups/86-videotulen/welcome).
 
+::: tip
+**Tip:** We nodigen iedereen uit feedback over ORI-A te delen via [onze GitHub!](https://github.com/Regionaal-Archief-Rivierenland/ORI-A-XSD)
+:::
```

Comment:
Ik denk dat goed is om bij een landelijk en publiek initiatief mensen actief te proberen te betrekken. Als we  aantrekken om github contribs te maken, dan wordt het beheer ook meer een community ding. Scheelt iedereen werk.  

Bovendien kunnen we  zo nieuwe beheerders scouten.